src/views/Transitioner.js

```diff
@@ -65,7 +65,7 @@ class Transitioner extends React.Component {
       this.state.position.removeListener(this._positionListener);
   }
 
-  componentWillReceiveProps(nextProps) {
+  UNSAVED_componentWillReceiveProps(nextProps) {
```

Comment:
This should be `UNSAFE`

```suggestion 
  UNSAFE_componentWillReceiveProps(nextProps) {
```